massa-models/src/denunciation_interest.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use crate::endorsement::WrappedEndorsement;
+use crate::operation::WrappedOperation;
+use crate::block::WrappedHeader;
+
+/// What can be sent to Denunciation factory (in order to create Denunciation)
+#[derive(Debug)]
+pub enum DenunciationInterest {
```

Comment:
As said this morning I believe this should be in `massa-factory-exports` but it's really no big deal if it stays here tbh

Reply:
I seems like it's gonna add quite some useless dependencies. Let's talk about that...

Reply:
Yeah after giving it some more thought I agree with you, since it would require to include the factory exports in modules that only need this struct